apps/api/src/app/auth/legacy-ee-auth/auth.controller.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  ClassSerializerInterceptor,
+  Controller,
+  Get,
+  Req,
+  Res,
+  UseGuards,
+  UseInterceptors,
+  Logger,
+} from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+
+import { ApiException, buildOauthRedirectUrl } from '@novu/application-generic';
+
+@Controller('/auth')
+@UseInterceptors(ClassSerializerInterceptor)
+export class AuthController {
+  @Get('/google')
```

Comment:
We can remove the google auth completely as it is not officially supported in the community edition. In fact we can remove the whole legacy-ee-auth folder.